web/public/mgmt/config.yml

```diff
@@ -1233,19 +1233,21 @@ collections:
   - name: "funding-opportunities"
     label: "Fund Opps - Content"
     folder: "content/src/fundings"
-    summary: "/{{urlSlug}} - {{name}}"
+    summary: "[{{agency[0]}}] - {{name}}"
```

Comment:
Never seen that syntax but played with it a bit and it seems neat.

Reply:
Would love it if we could somehow map to the agency name we typically use but I'm not sure if thats possible. So probably not worth the effort at the moment at least